stubs/tensorflow/tensorflow/train/__init__.pyi

```diff
@@ -20,16 +20,14 @@ from tensorflow.python.trackable.base import Trackable
 class CheckpointOptions:
     experimental_io_device: None | str
     experimental_enable_async_checkpoint: bool
-    # Uncomment when the stubs' TF version is updated to 2.15
-    # experimental_write_callbacks: None | list[Callable[[str], Any] | Callable[[], Any]]
+    experimental_write_callbacks: None | list[Callable[[str], Any] | Callable[[], Any]]
     enable_async: bool
 
     def __init__(
         self,
         experimental_io_device: None | str = None,
         experimental_enable_async_checkpoint: bool = False,
-        # Uncomment when the stubs' TF version is updated to 2.15
-        # experimental_write_callbacks: None | list[Callable[[str], Any] | Callable[[], Any]] = None,
+        experimental_write_callbacks: None | list[Callable[[str], Any] | Callable[[], Any]] = None,
```

Comment:
If return value does not matter for a callable prefer object over Any. Return argument is covariant so in practice it behaves same. Just that conventionally object as a return means anything is fine, Any when correct type is difficult to describe.

Reply:
Understood, done in 4dfca8278.